.github/workflows/ci.yml

```
@@ -162,6 +162,59 @@ jobs:
+          PR_REF=""
+          if [ -n "$PR_NUM" ]; then
+            PR_REF=" (from PR #${PR_NUM})"
+          fi
+
+          TITLE="CI broken on main after ${LAST_SHA:0:7}${PR_REF}"
+          read -r -d '' BODY <<ISSUEEOF || true
+## Problem
+
+CI failed on \`main\` after merging commit \`${LAST_SHA:0:10}\`: ${LAST_MSG}
+
+**Run:** ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)